gradient/cli/workflows.py

```diff
@@ -167,26 +167,30 @@ def get_workflow(api_key, workflow_id, show_runs, run, options_file):
     default=10000,
     cls=common.GradientOption,
 )
-@click.option(
-    "--follow",
-    "follow",
-    required=False,
-    default=False,
-    cls=common.GradientOption,
-)
+# TODO (roger): Disabled follow for now since we iterate through multiple workflow job ids 
```

Comment:
do we plan to reenable this?  (is there a follow up ticket or project?)

Reply:
yep, created a ticket internally